tcp_conn

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+# Expose total number of established TCP connections for the specified port.
+#
+# Usage: ./tcp_conn.sh -s|-d port 
+#
+# -s source port
+# -d destination port 
+# Example : add this to crontab:
+#
+# * * * * *   root tcp_conn.sh -d 8080 | sponge /var/lib/prometheus/node-exporter/tcp_conn.prom
+#
+# This will expose every minute total number of established TCP connections with `8080` as destination port 
+#
+# Author: Badreddin Aboubakr <badreddin.aboubakr@cloud.ionos.com>
+#         Daniel Swarbrick <daniel.swarbrick@cloud.ionos.com>
+
+set -e
+
+PATH=/usr/bin:/usr/sbin:/bin:/sbin
```

Comment:
Do not override the PATH in these scripts.

Reply:
Fixed :)